collectors/wt_client_collector.go

```diff
@@ -0,0 +1,86 @@
+package collectors
+
+import (
+	"context"
+	"fmt"
+	"encoding/hex"
+
+	"github.com/lightninglabs/lndclient"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// WtClientCollector is a collector that will export watchtower client related
+// metrics
```

Comment:
nit: we try to format comments as sentences (https://github.com/lightningnetwork/lnd/blob/master/docs/development_guidelines.md#code-documentation-and-commenting, so add a period)